quickannotator/api/v1/image/__init__.py

```diff
@@ -91,19 +92,67 @@ def get(self, args, project_id):
         """
         result = db.session.query(qadb.Image).filter(qadb.Image.project_id == project_id).all()
         return result, 200
+    
+@bp.route('/project/<int:project_id>', endpoint="images")
+class ImageSearch(MethodView):
+    @bp.arguments(SearchImageArgsSchema, location='query')
+    @bp.response(200, ImageRespSchema(many=True))
+    def get(self, args, project_id):
+        """     returns a list of Images
+        """
+        # images = db.session.query(qadb.Image).filter(qadb.Image.project_id == project_id).all()
+        # if images is not None:
+        #     return images
+        # else:
+        #     abort(404, message="Images not found")
+        images = db.session.query(
+            *[getattr(qadb.Image, column.name) for column in qadb.Image.__table__.columns],
+            db.func.ST_AsGeoJSON(qadb.Image.embedding_coord).label('embedding_coord')
+        ).filter(qadb.Image.project_id == project_id).all()
```

Comment:
if its easier, we can think about storing these as simple a tuple instead of a GEOM